src/Compute/Compute/Az.Compute.psd1

```diff
@@ -170,7 +170,8 @@ CmdletsToExport = 'Remove-AzAvailabilitySet', 'Get-AzAvailabilitySet',
                'Invoke-AzVmPatchAssessment', 
                'Get-AzDiskEncryptionSetAssociatedResource', 
                'Start-AzVmssRollingExtensionUpgrade', 'Invoke-AzVmInstallPatch', 
-               'Get-AzSshKey', 'Remove-AzSshkey', 'Update-AzSshKey', 'New-AzSshKey'
+               'Get-AzSshKey', 'Remove-AzSshkey', 'Update-AzSshKey', 'New-AzSshKey',
+                'New-AzDiskPurchasePlanConfig', 'New-AzDiskUpdateConfig', 'Set-AzDiskSecurityProfile'
```

Comment:
Last question:  what's for `New-AzDiskUpdateConfig`, are we able to name it as Update-AzDiskConfig?